### PR TITLE
build(aio): update examples to CLI to 1.6.5

### PR DESCRIPTION
**Pull request by Foxandxss**
Added a readme for how to update the CLI boilerplate. Updated and simplified the general README of the shared folder and removed a stray src/ folder.

### CODE REVIEW

**Comment by Foxandxss** [comment]
Yet another error (doesn't seem to be a flake) that only fails on travis

File: aio/tools/examples/README.md [hunk 1]
```diff
@@ -1,74 +1,65 @@
...
-  different tasks needed to run any example. Doesn't matter if CLI, System.js or Webpack.
-* **plnkr.json** - This file is used by the Plunker tool to generate a plunker for an example. This
-  concrete file is just a placeholder. Authors needs to tweak it for each guide. More at the
-  [plunker docs](../plunker-builder/README.md).
-* **example-config.json** - This file serves as a flag to indicate that the current folder is an
-  example. This concrete file is just a placeholder. More on this later in this readme.
+* CLI - For CLI based examples. This is the default one, to be used in the majority of the examples.
+* systemjs - Currently in deprecation, only used in a a few examples.
+* i18n - Based on the CLI one, features a few scripts for i18n.
+* universal - Based on the clie with a extra server for universal.
```
Comment (petebacondarwin) [inline aio/tools/examples/README.md]: clie -> cli

File: aio/tools/examples/README.md [hunk 2]
```diff
@@ -1,74 +1,65 @@
 # Overview
 
-Many of the documentation pages contain snippets of code examples. We extract these snippets from
+Many of the documentation pages contain snippets of code examples. Extract these snippets from
```
Comment (petebacondarwin) [inline aio/tools/examples/README.md]: I don't feel that removing the use of "we" makes this text clearer.

Comment (Foxandxss) [reply aio/tools/examples/README.md]: Wasn't the intention. I migrated it from "we" to "you" and I felt like I could simply not put "You" and still sounds good.

File: aio/tools/examples/README.md [hunk 3]
```diff
@@ -1,74 +1,65 @@
...
+
+There is also a `common` folder that contains files used in all different examples.
 
 ### The example-config.json
 
-So what is this **example-config.json** again? If an author wants to create a new example, say
-`/aio/content/examples/awesome-example`. The author needs to create an empty `example-config.json`
-in that folder. That serves as a flag so this tool will say "Hey, that is an example, let's copy
-all the boilerplate there".
+The **example-config.json** is what drives what kind of boilerplate you need for an example.
```
Comment (petebacondarwin) [inline aio/tools/examples/README.md]: I think we should avoid the chatty "Hey..." approach and say something like:

> Each example is identified by an example-config.json configuration file in its root folder.
> This configuration file indicates what type of boilerplate this example needs. E.g.
> ```
> { projectType: 'universal' }
> ```
> If the file is empty then the default type of `cli` is assumed.

File: aio/tools/examples/README.md (hunk 2 above)
Comment (petebacondarwin) [reply aio/tools/examples/README.md]: Not sure that "you" is better than "we", especially when it is really the tooling that is doing this for us.

**Comment by petebacondarwin** [comment]
@gkalpak pointed out to me that we also need to update the CLI version in the other boilerplate types: `i18n` and `testing`.

This leads us to consider whether we need to improve our boilerplate handling code to allow files to be "extended" rather than simply overriding. I am mostly in favour except that it adds yet more magic to our infrastructure. Also it would not be foolproof since it might be that we have extended something that still needs to be updated. But I guess that our CI testing will catch this.

File: aio/tools/examples/README.md (hunk 2 above)
Comment (Foxandxss) [reply aio/tools/examples/README.md]: Google rules of writing, I thought it would be better to follow them and more if other documents are using "You".

**Comment by IgorMinar** [comment]
I think the CI failure was just a flake. I'll try to restart it. @petebacondarwin do you have any concerns about landing this change?

**Review by petebacondarwin — APPROVED** [review]
Happy for this to land.

**Pull request closed without merging**